dts/bindings/ethernet/wiznet,w5500.yaml

```diff
@@ -23,3 +23,25 @@ properties:
       The reset pin of W5500 is active low.
       If connected directly the MCU pin should be configured
       as active low.
+  hw-reset-delay:
```

Comment:
seems like this is a software configuration that should not go in DT based on how it is used. maybe add another compatible to the node that indicates if the w5500 is the io variation?

Reply:
I had considered adding a compatible node for this variant, I will have another look into that soon